mbtb_app/web_portal/mbtb_portal/api/controllers/admin/revert-user-with-reason.js

```diff
@@ -25,12 +31,26 @@ module.exports = {
 
   fn: async function (inputs, exits) {
     let requests_ids = inputs.requests_ids;
+    let revert_reason = inputs.revert_reason;
     let payload = {
       suspend: "N",
+      revert_reason: revert_reason,
     };
 
-    // looping through received ids for tissue requests
+   /* for (i = 0; i < requests_ids.length; i++) {
```

Comment:
2 -> remove commented code and console log. Also add explanation in comment on line 44 (what part it does?).